pages/about.mdx

```diff
@@ -0,0 +1,68 @@
+import { LyonJSHead } from '../modules/header/LyonJSHead';
+import { Header } from '../modules/header/Header';
+import { Orgas } from '../modules/about/Orgas';
+import { Socials } from '../modules/about/Socials';
+
+<LyonJSHead />
+<Header />
+
+<main>
+
+<h1 className="text-sm text-gray-400 my-4">
+  A propos du Lyon JS : la communauté lyonnaise des utilisateurs de JavaScript
+</h1>
+
+## Qu&apos;est ce que le LyonJS ?
+
+Le LyonJS est un groupe de lyonnais·es qui a pour but de rassembler les utilisateur·rice·s web et JavaScript
+de la ville de Lyon et de ses alentours. Ainsi, nous organisons des réunions d&apos;informations mensuelles
```

Comment:
it could be cool if we ca can ride of `&apos;` in MDX in order to only be focus on the text. WYT ?
```suggestion
de la ville de Lyon et de ses alentours. Ainsi, nous organisons des réunions d'informations mensuelles
```